### ISSUE TITLE
Interval & Interval.Duration purpose

### ISSUE THREAD

**Issue by wclr** [purescript/purescript-datetime#92]
It is not clarified in docs  what is the purpose of `Data.Interval` and `Data.Interval.Duration`, in what cases it is supposed to used by design?

**Comment by garyb** [comment]
Yeah, it could use some doc comments. It's for what the names suggests - a time interval that can be repeated one or many times.

If I remember right it's modelled mostly after section 4.4 and 4.5 of ISO 8601:2004.
